a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -178,13 +178,14 @@ class ConfigureVirtEthIfaces(BaseVThunderTask):
     def execute(self, ve_interfaces, loadbalancer, vthunder):
         """Execute to configure ve on thunder device."""
         amphora_id = loadbalancer.amphorae[0].id
-        ve_segments = ve_interfaces[amphora_id]
-        c = self.client_factory(vthunder)
-        ve_client = acos_client.v30.interface.VirtualEthernet(c)
-        for segment, fixed_ip in six.iteritems(ve_segments):
-            netmask = '/' + fixed_ip.subnet.cidr.split('/')[1]
-            ve_client.update(segment, ip_address=fixed_ip.ip_address,
-                             ip_netmask=netmask)
+        if ve_interfaces:
```

Comment:
The case in which this doesn't exist should never occur. If it does occur we need to re-evaluate our flow.

Reply:
We added this because while adding member2 from same subnet , the `ve_interfaces` was coming None, hence we handled this case. However, we are revisiting this part while testing all scenarios and will fix it.